src/os/bluestore/BlueStore.cc

```diff
@@ -6046,14 +6046,13 @@ void BlueStore::_txc_state_proc(TransContext *txc)
         sb->bc.finish_write(txc->seq);
       }
       txc->shared_blobs_written.clear();
+      _txc_finalize_kv(txc, txc->t);
```

Comment:
@liewegas .  I see https://trello.com/c/YJZymq8O/8-bluestore-fix-and-test-inline-parallel-txn-submission also related parallel submit for Allocator. But i don't know why for extentfreelist dose not support paralle? Because the global lock or other reasons?  Thanks!